src/schema/me/bidder_position_mutation.js

```diff
@@ -1,10 +1,70 @@
 // @ts-check
 
-import { GraphQLString, GraphQLFloat, GraphQLNonNull } from "graphql"
+import { GraphQLString, GraphQLFloat, GraphQLObjectType, GraphQLNonNull } from "graphql"
 import { mutationWithClientMutationId } from "graphql-relay"
 
 import BidderPosition from "schema/bidder_position"
 
+const biddingErrors = [
+  {
+    id: "ERROR_BID_LOW",
+    gravity_key: "Please enter a bid higher than",
+    header: "Your bid wasn't high enough",
+    description: `Another bidder placed a higher max bid or the same max bid before you did.\
+ Bid again to take the lead.`,
+  },
+  {
+    id: "ERROR_SALE_CLOSED",
+    gravity_key: "Sale Closed to Bids",
+    header: "Lot closed",
+    description: "Sorry, your bid wasn’t received before the lot closed.",
+  },
+  {
+    id: "ERROR_LIVE_BIDDING_STARTED",
+    gravity_key: "Live Bidding has Started",
+    header: "Live bidding has started",
+    // TODO link
+    description: `Sorry, your bid wasn’t received before live bidding started.\
+ To continue bidding, please join the live auction.`,
+  },
+  {
+    id: "ERROR_BIDDER_NOT_QUALIFIED",
+    gravity_key: "Bidder not qualified to bid on this auction.",
+    header: "Bid not placed",
+    // TODO link
+    description: `Your bid can’t be placed at this time.\
+ Please contact support@artsy.net for more information.`,
+  },
+  {
+    id: "ERROR_UNKNOWN",
+    gravity_key: "unknown error",
+    header: "Bid not placed",
+    // TODO link
+    description: `Your bid can’t be placed at this time.\
+ Please contact support@artsy.net for more information.`,
+
+  },
+]
+
+const BidderPositionMutationResultType = new GraphQLObjectType({
+  name: "BidderPositionMutationResult",
+  fields: () => ({
+    status: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    message_header: {
+      type: GraphQLString,
+    },
+    message_description: {
+      type: GraphQLString,
+    },
+    position: {
```

Comment:
Oh nice solution to include the bidder position here so you don't have to make up an ID!